content/en/roadmap.md

```diff
@@ -183,3 +187,16 @@ Tasks
 - [x]  <span style="color:grey">Implement support for referring to an alternative chart values file</span>
 - [x]  <span style="color:grey">Stabilize API</span>
 - [x]  <span style="color:grey">[Create a migration guide for Helm Operator users](docs/migration/helm-operator-migration.md)</span>
+
+## Flagger
+
+### [GitOps Toolkit](https://github.com/fluxcd/flux2) compatibility
+
+- [ ] Migrate Flagger to Kubernetes controller-runtime and [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder)
+- [ ] Make the Canary status compatible with [kstatus](https://github.com/kubernetes-sigs/cli-utils)
```

Comment:
These two points don't fall under GOTK compatibility, I think the "Integrations" section might be more apt.

Reply:
I simply copied from https://github.com/fluxcd/flagger/blob/main/README.md#roadmap :grimacing:

Reply:
oh I see, well that's wrong then I guess. I don't see how `kubebuilder` usage and `kstatus` compatibility fall under GOTK.
Maybe we should have another section named "Kubernetes Ecosystem" for these two?

Reply:
@aryan9600 kstatus compatibility is needed for Flux to be able to wait for Canaries.